com.yakindu.sct.examples.stm32f407vg.blinky/Core/Inc/sc_timer_service.h

```diff
@@ -1,71 +0,0 @@
-/*
```

Comment:
``sc_timer_service.h`` is not a generated file! Don't remove it.

Reply:
restored